classification/classification_network.py

```diff
@@ -322,8 +321,10 @@ def createNetwork(args):
     dataSource   (RecordStream) Sensor region reads data from here.
     sensorType   (str)          Specific type of region, e.g. "py.RecordSensor";
                                 possible options can be found in nupic/regions/.
+    classifierType   (str)      Specific type of classifier region, e.g. "py.CLAClassifier";
```

Comment:
We'll make the argument handling cleaner in the future, but for now, to minimize confusion, please update the docstring to reflect the exact tuple that needs to be passed in.